packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
```

Comment:
Rendering this components it gives an hydration error. You can check by refreshing the eth components page

This happens because of the wagmi cache. To avoid it, you need to use an extra useState. Check `Address.tsx`

![image](https://user-images.githubusercontent.com/2486142/194177027-3f0df284-ea19-45e5-ab5f-74fa7921dd14.png)

Reply:
for this have another solution like this 
`  const [mounted, setMounted] = useState(false);

  // default nextjs hydrtion issue managed
  useEffect(() => setMounted(true), []); // at init only
  if (!mounted) {
    return null;
  }

`

if we add this code inside _app.jsx then hydration issue will resolve . 
and we don't have to add useState pattern for wagmi  on every component . 

let me know if it is ok? 
will raise another pr for this fix